### PR TITLE
Refactor the Person class and relevant fields

### PR DESCRIPTION
**Pull request by andrewtanJS**
The main changes are as follows:
Person -> Student
Phone -> StudentNumber
Address -> Faculty
The other many changes follow from these.

#77

### CODE REVIEW

**Comment by andrewtanJS** [comment]
@yogtew where sepcifically so i may change them?

**Comment by yogtew** [comment]
> @yogtew where sepcifically so i may change them?

Okay I will comment in the files where it needs to be changed.

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -157,78 +157,78 @@ Examples:
...
 
 Examples:
 
 * `find John` +
 Returns `john` and `John Doe`
 * `find Betsy Tim John` +
-Returns any person having names `Betsy`, `Tim`, or `John`
+Returns any student having names `Betsy`, `Tim`, or `John`
 * `find t/students` +
 Returns people tagged with `t/students`
```
Comment (yogtew) [inline docs/UserGuide.adoc]: People -> students

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -157,78 +157,78 @@ Examples:
...
 * `find t/students t/colleagues` +
 Returns people tagged with `t/students` or `t/colleagues`
```
Comment (yogtew) [inline docs/UserGuide.adoc]: People -> students

File: src/main/java/seedu/address/logic/commands/mark/MarkFindCommand.java [hunk 3]
```diff
@@ -7,22 +7,22 @@
...
 import seedu.address.model.Model;
 import seedu.address.model.mark.Mark;
-import seedu.address.model.person.Person;
+import seedu.address.model.student.Student;
 
 /**
  * Mark "find" subcommand
  */
 public class MarkFindCommand extends MarkCommand {
     private static final String MESSAGE_SUCCESS = "Successfully added %d people to %s";
```
Comment (yogtew) [inline src/main/java/seedu/address/logic/commands/mark/MarkFindCommand.java]: People -> students

**Comment by andrewtanJS** [comment]
The student number has been modified to accept only alphanumeric values, and the refactoring has been done for the remainder of the tests.

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -56,43 +56,43 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 
 Format: `help`
 
-=== Adding a person: `add`
+=== Adding a student: `add`
 
-Adds a person to the address book +
+Adds a student to the address book +
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
```
Comment (yogtew) [inline docs/UserGuide.adoc]: Format still has PHONE_NUMBER

**Pull request merged**